services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/controller/SubnetController.java

```diff
@@ -477,6 +477,10 @@ public ResponseId deleteSubnetState(@PathVariable String projectId, @PathVariabl
         } catch (ParameterNullOrEmptyException e) {
             logger.error(e.getMessage());
             throw new Exception(e);
+        } catch (HavePortInSubnet e) {
```

Comment:
We will need a better handling of exceptions. 
- Evaluate whether or not we will need to log information for the caught exception. 
- If logging is an option, we will like to include useful information in the message (in this case, subnet id etc.)
- If not, we could combine exceptions in one line, for example, 
```
catch(ParameterNullOrEmptyException | HavePortInSubnet e)
{
...
}
```

Reply:
Sure

Reply:
I checked the code and found that it was not good to log the info, so I combined exceptions in one line.